sdk/bom/azure-sdk-bom.xml

```diff
@@ -0,0 +1,120 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-sdk-bom</artifactId>
+  <version>1.0.0-beta.1</version>
```

Comment:
Should this be 1.0.0 instead? I don't think we are releasing beta versions for BOM.

Reply:
Just left as 1.0.0-beta.1 for now, will update when being released.